.github/ISSUE_TEMPLATE/requirement.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Requirement
+about: Use this template to propose an LWS requirement.
+title: "[REQ] <<brief description of requirement>>"
+labels: triage
+
+---
+
+**A [component],**
+**[MUST,SHOULD,MAY...] <<description or requirement>>**
```

Comment:
* as suggested, I believe that normative language must not be used here
* the proposal below also aims at giving a little more guidance (my 2¢)
```suggestion
[component / actor] shall [behave like this | have that property]
```

Reply:
This suggestion is a significant improvement to the original proposal, and better aligned with the way UCR documents are expressed.

The key is to make sure that the way the requirements are expressed, they are not merely solutions engineering. It'll be obvious when we review the UC requirements any way. So, the template should be a reasonably useful guide to enable multiple solutions.